lib/backports/2.6.0/method/compose.rb

```diff
@@ -0,0 +1,13 @@
+unless Method.method_defined?(:<<) || Method.method_defined?(:>>)
+  require 'backports/2.6.0/proc/compose'
+
+  class Method
+    def <<(g)
+      to_proc << g
+    end
```

Comment:
Best to put the `unless Method.method_defined?(:<<)` here...

Reply:
I completely disagree with this.  If one of the operators is already defined, defining the paired operator to potentially be a completely different operation doesn't make a lot of sense.  To me it only makes sense to define paired operators if neither of them is already defined.

Reply:
Ok